openpgp/packet/symmetric_key_encrypted.go

```diff
@@ -117,16 +196,34 @@ func SerializeSymmetricKeyEncrypted(w io.Writer, passphrase []byte, config *Conf
 	}
 	s2kBytes := s2kBuf.Bytes()
 
-	packetLength := 2 /* header */ + len(s2kBytes) + 1 /* cipher type */ + keySize
+	var packetLength int
+	if v5 {
+		nonceLen := config.AEADConfig.Mode().NonceLength()
+		tagLen := config.AEADConfig.Mode().TagLength()
+		packetLength = 3 + len(s2kBytes) + nonceLen + keySize + tagLen
```

Comment:
Hmm if it were for symmetry, it would be a `3 /* header */` (note that the `+1 /* cipher type */` is included on the first byte of the key, and it's not there for v5)

Reply:
Oh right, my bad, I thought it was referring to the same thing. Never mind, then.